pkg/config/config.go

```diff
@@ -97,22 +97,24 @@ func (c *Config) watch() {
 	configFiles := c.configFiles()
 	err := e.configFilesChangedObserver.Watch(configFiles)
 	if err != nil {
-		glog.Errorf("watch [Error]: %v", err)
+		glog.Error("Can't watch configuration files: %s", err)
```

Comment:
'%s' ?

Reply:
I think that error messages should be included without quotes, as they may contain quotes themselves.

Reply:
:+1: